src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -644,6 +644,63 @@ public void TestGenerateEvent_From_AuthFlowResult_With_TokenResult_And_Null_Erro
             eventData.Measures.Should().ContainKey("duration_milliseconds");
         }
 
+        /// <summary>
+        /// Test to validate user interactive env.
+        /// </summary>
+        [Test]
+        public void InteractivityIsNotDisabledIfCorextIsSetToOne()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            this.envMock.Setup(e => e.Get("Corext_NonInteractive")).Returns("1");
+            subject.InteractivityDisabled().Should().BeTrue();
+        }
+
+        /// <summary>
+        /// Test to validate user interactive env.
+        /// </summary>
+        [Test]
+        public void InteractivityIsNotDisabledIfCorextIsNotSetToOneOrTrue()

```

Comment:
```suggestion
        public void InteractivityIsEnabledIfCorextIsNotSetToOneOrTrue()

```

Reply:
Prefer in the naming to only use "IsEanbled" or "IsDisabled" to make it easier to read.